src/utils.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+import type { LocaleType } from '@paypal/sdk-constants/src';
+
+const SPECIAL_COUNTRY_LANG_VARIANTS = {
+    'HK': 'zh_Hant',
+    'TW': 'zh_Hant'
+};
```

Comment:
I worry about maintaining another list here. What do you think about reducing this `COUNTRY_LANGS` from sdk-constant to get all the countries that support `zh_Hant`? 

https://github.com/paypal/paypal-sdk-constants/blob/master/src/locale.js#L460

Reply:
That seems like a more maintainable approach to me as well. Then we can maintain the list in a singular spot which could prove useful later.

Reply:
I was doing some tests with that approach, we change the logic a little bit, in this new case, if the list of languages for the country includes `zh_Hant` , then return it as lang ... what do u think?

```
/* @flow */
import type { LocaleType } from '@paypal/sdk-constants/src/';
import { COUNTRY_LANGS } from '@paypal/sdk-constants/src';


export function getComputedLocales(locale : string) : LocaleType {
    let [ lang, country ] = locale.split('_');
    // $FlowFixMe
    const countryLangs = COUNTRY_LANGS[country];

    if (countryLangs.includes('zh_Hant')) {
        lang = 'zh_Hant';
    }


    // $FlowFixMe
    return { lang, country };
}
```

Reply:
I like this approach! I left a couple comments. 

Can you also add a test for this use case?

Reply:
Sure, added!